CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 
 #### New Features
 * Fix #1917: Allow user to add custom Headers to client
+* Fix #1948: LeaderElection Implementation (Lock Implementations: ConfigMap & Lease)
```

Comment:
Shouldn't it be in 4.7-SNAPSHOT section?

Reply:
:man_facepalming: arrg, I just fixed the conflict using the web interface and I didn't notice/forgot  we had a "new section" since last release on friday. Chaing

Reply:
Changed, I should have known better, never use GitHub's web interface :nauseated_face: